staging/src/k8s.io/kubelet/config/v1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// +k8s:openapi-gen=true
+// +groupName=kubelet.config.k8s.io
+
+package v1 // import "k8s.io/kubelet/config/v1"
```

Comment:
This seems to be the first v1 API. Is it now a requirement that all GA features need to migrate their APIs to v1? And I guess this means we plan to migrate the existing GA features' APIs to v1

Reply:
Since this feature has a dedicated top-level Config type, and a serialized request/response type send to/from the credential plugin, yes, those API types must be v1 for the feature to be GA